### PR TITLE
updated e2e to check Member has apiEndpoint

### PR DESCRIPTION
**Pull request by ranakan19**
wrt fix: https://issues.redhat.com/browse/CRT-971
changes in host-operator in : https://github.com/codeready-toolchain/host-operator/pull/394

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -43,6 +43,7 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
+	github.com/codeready-toolchain/toolchain-common => github.com/ranakan19/toolchain-common v0.0.0-20210318182646-b0141bf2d375 // using 'github.com/ranakan19/codeready-toolchain@CRT971
```
Comment (alexeykazakov) [inline go.mod:46]: Just a reminder.

**Review by alexeykazakov — APPROVED** [review]
Looks good! But don't forget to remove the replace before merging :)

**Review by MatousJobanek — APPROVED** [review]
:+1:

**Comment by ranakan19** [comment]
closing this PR as the e2e test wouldn't pass because of branch name CRT971 being present in all three forks of host-operator, member-operator and registration-service.

@alexeykazakov and @MatousJobanek : If you could please approve the new [PR](https://github.com/codeready-toolchain/toolchain-e2e/pull/267), I can get this merged.

**Pull request closed without merging**